df.creature-raws.xml

```diff
@@ -1032,38 +1105,42 @@
         </compound>
 
         <stl-vector name='secretion'>
-            <pointer>
+            <pointer> bay12: secretion_profilest
                 <int16_t name='mat_type' ref-target='material' aux-value='$$.mat_index'/>
                 <int32_t name='mat_index'/>
                 <enum type-name='matter_state' base-type='int16_t' name='mat_state'/>
-                <stl-string name="mat_type_str"/>
-                <stl-string name="mat_index_str"/>
-                <stl-string name="unk_44"/>
+                <stl-string name="mat_str1"/>
+                <stl-string name="mat_str2"/>
+                <stl-string name="mat_str3"/>
                 <stl-vector name="body_part_id" type-name="int16_t" refers-to='$$._upglobal.body_info.body_parts[$]'/>
                 <stl-vector name="layer_id" type-name="int16_t" refers-to='$$._parent._parent.body_part_id[$$._key].refers-to.layers[$]'/>
-                <int32_t name='cause' since='v0.40.01' comment='2 EXERTION, 1 EXTREME_EMOTION, 0 always?'/>
+                <enum name='cause' base-type='int32_t'> bay12: SecretionCondition
+                    <enum-item name='CONTINUOUS'/>
+                    <enum-item name='EXTREME_EMOTION'/>
+                    <enum-item name='EXERTION'/>
+                </enum>
             </pointer>
         </stl-vector>
 
         <stl-vector name='creature_class' pointer-type='stl-string'/>
 
-        <compound name='unknown2'>
+        <compound name='unknown2'> not actually a compound
```

Comment:
since the name isn't useful, would it be too disruptive to just get rid of the compound?

Reply:
We can worry about removing unneeded compounds later - right now I'm trying to minimize changes (aside from ones that are absolutely necessary).